Curs/src/style/login.less

```diff
@@ -169,36 +228,108 @@ a {
     }
 
     .right-image {
-        float: left;
+        margin-bottom: 50px;
+        margin-top: 200px;
+        width: 100%;
+        margin-left: 0;
     }
 }
 
 /*Screen between 699px and 768px*/
 @media screen and (max-width: 768px) and (min-width: 699px) {
     /* Styles */
 
+    #alert {
+        color: red;
+        font-size: 20px;
+        width: 97%;
+    }
+    
+    .right-text-description {
+        width: 100%;
+        margin-top: 350px;
+        text-align: center;
+        float: none;
+    }
+
     .footer-section {
         text-align: center;
     }
 
     .feedback-image {
-        float: left !important;
+        width: 60%;
+        margin-left: 150px;
+        height: 300px;
     }
 
     .header-form-section {
         width: 57%;
         margin: 0 auto;
     }
 
-    .right-image {
+    .text-description-2 {
+        margin-top: 0;
+    }
+
+    .left-text-description {
+        width: 100%;
+        margin-top: 100px;
+        text-align: center;
         float: left;
+        padding-left: 0;
+    }
+
+    .right-image {
+        margin-bottom: 50px;
+        margin-top: 200px;
+        width: 100%;
+        margin-left: 0;
     }
+
 }
 
 /*Screen between 768px and 992px*/
 @media screen and (max-width: 992px) and (min-width: 768px) {
     /* Styles */
 
+    #alert {
+        color: red;
+        font-size: 20px;
+        width: 97%;
+    }
+
+    .feedback-image {
+        width: 60%;
+        margin-left: 150px;
+        height: 300px;
+    }
+
+    .right-text-description {
+        width: 100%;
+        margin-top: 350px;
+        text-align: center;
+        float: none;
+    }
+
+    .left-text-description {
+        width: 100%;
+        margin-top: 100px;
+        text-align: center;
+        float: left;
+        padding-left: 0;
+    }
+
+    .text-description-2 {
```

Comment:
Can you 'splain to me why the trailing numeric value?

Reply:
That was my best idea at the moment. :)

I changed with "text-description-details". I hope that this choice is better. :)